examples/metalk8s-solution-example/example-operator/pkg/controller/example/example_controller.go

```diff
@@ -227,17 +225,19 @@ func (r *ReconcileExample) deploymentForExample(example *solutionv1alpha1.Exampl
 	return deployment
 }
 
-func labelsForExample(example *solutionv1alpha1.Example) map[string]string {
-	return map[string]string{
+func labelsForExample(example *solutionv1alpha1.Example, versionize bool) map[string]string {
+	var labels = map[string]string{
 		"app":                                   "example",
-		"app.kubernetes.io/name":                "example",
-		"app.kubernetes.io/version":             example.Spec.Version,
+		"app.kubernetes.io/name":                example.Name,
 		"app.kubernetes.io/component":           "component",
-		"app.kubernetes.io/part-of":             "example-solution",
+		"app.kubernetes.io/part-of":             "example",
 		"app.kubernetes.io/managed-by":          "example-operator",
-		"example.solution.com/cr-name":          example.Name,
-		"example.solution.com/operator-version": version.Version,
 	}
+	if versionize {
+		labels["app.kubernetes.io/version"] = example.Spec.Version
+		labels["example-solution@scality.com/operator-version"] = version.Version
```

Comment:
Labels are namespaced by domain, not e-mail address. Also, keep this namespace consistent with the `apiVersion`.